src/components/data/LocalVariablesContent.js

```diff
@@ -18,13 +18,31 @@ import * as StateHistory from 'src/libs/state-history'
 import * as Utils from 'src/libs/utils'
 
 
+export const getDisplayedAttribute = arr => {
+  const { key, value, description } = _.assign(...arr, {})
+  return [key, value, description ?? '']
+}
+
+const DESCRIPTION_SUFFIX = '__DESCRIPTION__'
+const isDescriptionKey = k => k.endsWith(DESCRIPTION_SUFFIX)
+
+export const renameAttribute = ([k, v]) => isDescriptionKey(k) ?
+  {
+    key: k.substring(0, k.length - DESCRIPTION_SUFFIX.length),
+    description: v
+  } : {
+    key: k,
+    value: v
+  }
```

Comment:
Perhaps a bit easier?
```suggestion
export const renameAttribute = ([k, v]) => isDescriptionKey(k) ?
  { key: k.slice(0, -DESCRIPTION_SUFFIX.length), description: v } :
  { key: k, value: v }
```

Reply:
Done.